model_compiler/python/nnir.py

```diff
@@ -447,13 +447,24 @@ def updateLocals(self):
                     self.addLocal(local)
                 elif node.type in ['slice']:
                     input = self.tensor_dict[node.inputs[0]]
-                    shape = [input.shape[0], input.shape[1] // len(node.outputs), input.shape[2], input.shape[3]]
-                    for name in node.outputs:
-                        local = IrTensor()
-                        local.setName(name)
-                        local.setInfo(input.type, shape)
-                        local.setFormat(input.format)
-                        self.addLocal(local)
+                    starts = np.frombuffer(self.binaries[node.inputs[1]], dtype=np.int32)
+                    ends = np.frombuffer(self.binaries[node.inputs[2]], dtype=np.int32)
```

Comment:
Codacy found an issue: [local variable 'ends' is assigned to but never used (F841)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=5932170)